sdk-setup/src/mb2

```diff
@@ -2343,8 +2343,8 @@ sign_rpms__prepare() {
 }
 
 sign_rpms() {
-    [[ -e $STATEDIR/spec ]] || fatal "No previous build found"
-    local rpms=$(find_rpms $all_opt "$OPT_OUTPUTDIR" "$OPT_PKGDIR" "$STATEDIR/spec")
+    [[ -e $TMP_SPEC ]] || fatal "No previous build found"
```

Comment:
Just wondering if we should pass the spec file as argument for these functions as well

Reply:
yup, fixed, thanks